src/components/Comment/Comment.tsx

```diff
@@ -291,7 +283,13 @@ export const Comment = ({
                 ) : (
                   <></>
                 )}
-                <div className={timestampWrapperStyles}>
+                <div
+                  className={cx(
+                    timestampWrapperStyles,
+                    marginLeftStyles,
+                    flexColStyles
```

Comment:
All of these abstractions are only ever used here so we can simplify this a lot by just having `timstampWrapperStyles` - there's no need for the other variables